test/e2e/common_test.go

```diff
@@ -622,7 +784,7 @@ func (p *PodmanTestIntegration) RunTopContainer(name string) *PodmanSessionInteg
 // runs top.  If the name passed != "", it will have a name, command args can also be passed in
 func (p *PodmanTestIntegration) RunTopContainerWithArgs(name string, args []string) *PodmanSessionIntegration {
 	// In proxy environment, some tests need to the --http-proxy=false option (#16684)
-	var podmanArgs = []string{"run", "--http-proxy=false"}
+	podmanArgs := []string{"run", "--http-proxy=false"}
```

Comment:
here as well please avoid all formatting changes like this. IT makes it harder to review.

Reply:
Reverted